src/BPINN_ode.jl

```diff
@@ -148,21 +148,36 @@ end
 BPINN Solution contains the original solution from AdvancedHMC.jl sampling(BPINNstats contains fields related to that)
 > ensemblesol is the Probabilistic Estimate(MonteCarloMeasurements.jl Particles type) of Ensemble solution from All Neural Network's(made using all sampled parameters) output's.
 > estimated_nn_params - Probabilistic Estimate of NN params from sampled weights,biases
-> estimated_ode_params - Probabilistic Estimate of ODE params from sampled unknown ode paramters
+> estimated_de_params - Probabilistic Estimate of DE params from sampled unknown de paramters
 """
 struct BPINNsolution{O <: BPINNstats, E,
-    NP <: Vector{<:MonteCarloMeasurements.Particles{<:Float64}},
+    NP <:
+    Union{
+        Vector{<:MonteCarloMeasurements.Particles{<:Float64}},
+        Vector{
+            <:ComponentArrays.ComponentVector{
+                <:MonteCarloMeasurements.Particles{<:Float64},
+            },
+        },
+        Vector{<:Vector{<:MonteCarloMeasurements.Particles{<:Float64}}
+        },
+    },
     OP <: Union{Vector{Nothing},
-        Vector{<:MonteCarloMeasurements.Particles{<:Float64}}}}
+        Vector{<:MonteCarloMeasurements.Particles{<:Float64}}}, P}
     original::O
     ensemblesol::E
     estimated_nn_params::NP
-    estimated_ode_params::OP
-
-    function BPINNsolution(original, ensemblesol, estimated_nn_params, estimated_ode_params)
+    estimated_de_params::OP
+    points::P
```

Comment:
What are the points here?

Reply:
The Domain points got using the domains and saveat vector for grid spacings for corresponding Var, at which the ensemble solution is evaluated

Reply:
Let's make the name more descriptiove, timepoints?